### ISSUE TITLE
Keyboard navigation does not work for inline or custom input

### ISSUE THREAD

**Issue by benbcai** [Hacker0x01/react-datepicker#744]
It looks like you cannot navigate in the picker using the left/right/up/down keys for inline or with a custom input. See the inline and custom input examples in https://hacker0x01.github.io/react-datepicker/.

**Comment by amit-coditas** [comment]
+1

**Comment by benbcai** [comment]
@amit-coditas I still cannot get keyboard navigation to work for the inline picker and I think the reason is because there is no input to receive the onKeyDown event. However, I was able to get keyboard navigation to work for my custom input. I just had to add onKeyDown={this.props.onKeyDown} to my custom input.

**Comment by kristofferahl** [comment]
Also not getting keyboard navigation to work when using the inline version. Would really like a fix to this. Awsome component btw!

**Comment by DavidHooper** [comment]
@benbcai had the solution for me. I am using a custom input.

**Comment by alex-shamshurin** [comment]
I'm interested in that too.
